tests/db/message_test.py

```diff
@@ -0,0 +1,100 @@
+# encoding=utf-8
+# Copyright © 2017 Dylan Baker
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import
+import unittest
+
+import mock
+
+from notmuch import NullPointerError
+
+from alot import account
+from alot.db import message
+
+
+class MockNotmuchMessage(object):
+    """An object that looks very much like a notmuch message.
+
+    All public instance variables that are not part of the notmuch Message
+    class are prefaced with mock.
+    """
+
+    def __init__(self, headers=None):
+        self.mock_headers = headers or {}
+        self.mock_message_id = 'message id'
+        self.mock_thread_id = 'thread id'
+        self.mock_date = 0
+        self.mock_filename = 'filename'
+        self.mock_tags = ['example']
+
+    def get_header(self, field):
+        return self.mock_headers.get(field, '')
+
+    def get_message_id(self):
+        return self.mock_message_id
+
+    def get_thread_id(self):
+        return self.mock_thread_id
+
+    def get_date(self):
+        return self.mock_date
+
+    def get_filename(self):
+        return self.mock_filename
+
+    def get_tags(self):
+        return self.mock_tags
+
+
+class TestMessage(unittest.TestCase):
+
+    def test_get_from_email_only(self):
+        """From is populated using the 'From' header when only an email address
```

Comment:
shouldn't this docstring tell us what the method tests? same for the other docstrings in this file..